sdx-submitter.go

```diff
@@ -0,0 +1,138 @@
+package main
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"github.com/streadway/amqp"
+	"io/ioutil"
+	"os"
+)
+
+func main() {
+	// Read a source file and place it on a rabbit topic exchange
+	// Exchange , Queue and binding must be in place before use
+
+	var url, exchange, routingKey, encryptionKey, signingKey, name, password string
+	var port, host, queue, vhost, messageFilePath string
+	var msgBody []byte
+	var err error
+	const PRINT_MSG_CHAR_COUNT = 20 // only print the first few characters of the message
```

Comment:
Consts should be named camelCase as with any other var (idiomatic Go). If they're private to the package, then starting with a lowercase, otherwise starting with Uppercase if exported.

Should declare consts in the package outside of main where possible

Reply:
fair 'nuff . I just slipped back into C . It flows nicer to my eyes if consts are obviously consts , but then again I used to like polish notation . will change and push